packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -150,6 +173,9 @@ class RawAutocomplete<T extends Object> extends StatefulWidget {
   ///
   /// Pass the provided [TextEditingController] to the field built here so that
   /// RawAutocomplete can listen for changes.
+  ///
+  /// If this parameter is null, then a [SizedBox.shrink] is built instead.
+  /// For how that pattern can be useful, see [textEditingController].
```

Comment:
Cool, good to get this feature written down in the docs.

Oh hmm but this is inside the `{@template …}`/`{@endtemplate}` block. I think that means it'll show up on the [Autocomplete.fieldViewBuilder] doc as well. It doesn't belong there, right?

In that case I think a fix is to just put this paragraph after the `{@endtemplate}`, instead of before.